### PR TITLE
Completed unit-tests-jest - Timothy Lee

### PR DESCRIPTION
**Pull request by timlee712**
unit-tests-jest

### CODE REVIEW

File: unit-tests-jest/numbers.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers.js';
+
+describe('evenNumbers()', () => {
+  it('should return an array containing only even numbers', () => {
+    const numbers = [1, 4, 5, 10, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([4, 10, 0]);
+  });
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.test.js:8]: There are a number of edge cases you could test, such as an empty array, non-integer values, etc.

File: unit-tests-jest/numbers.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+});
+
+describe('multiplyBy', () => {
+  it('modifies the object by multiplying each number value by the multiplier', () => {
+    const obj = { a: 5, b: 'hello', c: 7 };
+    const multiplier = 3;
+    const result = multiplyBy(obj, multiplier);
+    expect(result).toEqual({ a: 15, b: 'hello', c: 21 });
+    expect(obj).toEqual({ a: 15, b: 'hello', c: 21 });
+  });
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.test.js:35]: And here.
